### PR TITLE
Update .npmignore

### PR DESCRIPTION
**Pull request by peakji**
Corrected leveldb and snappy versions.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -6,11 +6,11 @@ test-data.db.tar
 deps/leveldb/leveldb-basho/
 deps/leveldb/leveldb-hyper/
 deps/leveldb/leveldb-rocksdb/
-deps/snappy/snappy-1.1.1/testdata/
+deps/snappy/snappy-1.1.4/testdata/
```
Comment (ralphtheninja) [inline .npmignore]: Nice catch. This removes about 2.9 Mb.

Comment (vweevers) [inline .npmignore]: How about a wildcard (`snappy-*`)?

Comment (peakji) [reply .npmignore]: Thanks! ➡️ https://github.com/Level/leveldown/pull/412/commits/ddee44685b38d1a5353555e1ba078ff5880208b3

**Comment by ralphtheninja** [comment]
2.1.1

**Pull request merged**